yt_dlp/extractor/zdf.py

```diff
@@ -305,23 +344,38 @@ def _extract_entry(self, url, player, content, video_id):
             'title': chap.get('anchorLabel'),
         } for chap, next_chap in zip(chapter_marks, chapter_marks[1:])]
 
+        target_info = traverse_obj(content, ('programmeItem', 0, 'http://zdf.de/rels/target'))
         return merge_dicts(info, {
             'title': title,
             'description': content.get('leadParagraph') or content.get('teasertext'),
             'duration': int_or_none(t.get('duration')),
             'timestamp': unified_timestamp(content.get('editorialDate')),
             'thumbnails': thumbnails,
             'chapters': chapters or None,
+            'seriesId': traverse_obj(target_info, ('http://zdf.de/rels/cmdm/series', 'seriesUuid')),
```

Comment:
```suggestion
            'series_id': traverse_obj(target_info, ('http://zdf.de/rels/cmdm/series', 'seriesUuid')),
```

Reply:
(also did this change in the tests to make sure they still pass)